README.md

```diff
@@ -99,6 +99,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
     - [Podcasts](#podcasts)
     - [Websites](#websites)
 - [Contributing](#contributing)
+-http://rgruet.free.fr/#QuickRef (# my mini Python tutorial)
```

Comment:
Does not belong here. I am not going to allege this is a try to get the free t-shirt, if not it could be a content for "websites" at the bottom.